changelog.config.js

```diff
@@ -0,0 +1,88 @@
+const COMPONENT_SPECS = require('genesys-spark-components/build/component-specs/component-specs.json');
+const CHART_COMPONENT_SPECS = require('genesys-spark-chart-components/build/component-specs/component-specs.json');
+
+const componentSpecs = Object.assign(COMPONENT_SPECS, CHART_COMPONENT_SPECS);
+
+const getComponentSpec = function () {
+  let scopeList = ['other'];
+  for (const [key] of Object.entries(componentSpecs)) {
+    scopeList.push(key);
+  }
+  return scopeList;
+};
+
+module.exports = {
+  disableEmoji: true,
+  format: '{type}{scope}: {emoji}{subject}',
+  list: [
+    'test',
+    'feat',
+    'fix',
+    'chore',
+    'docs',
+    'refactor',
+    'style',
+    'ci',
+    'perf'
```

Comment:
In the current ci, there's a commit type 'build'. Is there a conscious decision to exclude that here?

Reply:
I'm just leveraging the default `git-cz` config here, I think `build` is covered under `chore:      Build process or auxiliary tool changes`